modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc

```diff
@@ -25,13 +23,13 @@ nto-payments   customers   anypoint-service-mesh-adapter   1234    customers
 
 == Methods for Binding an API
 
-You can bind an API to a service using Anypoint Platform or Anypoint Service Mesh, based on your preference:
+You can bind an API to a service using API ID or Anypoint Service Mesh API name, based on your preference:
 
-* <<bind_the_service, Bind an API created using Anypoint Platform>>: If you created your API using Anypoint Platform, use this procedure.
-* <<bind_using_ASM, Bind an API created using Anypoint Service Mesh>> : If you created your API using Anypoint Service Mesh, use this procedure.
+* <<bind_the_service, Bind an API referencing an API ID>>:  Use this procedure if you want to bind to the API using the API ID. This way is recommended if you created your API using Anypoint Platform.
+* <<bind_using_ASM, Bind an API created using Anypoint Service Mesh API name>>: Use this procedure if you want to bind to the API using the Service Mesh API name. This way is recommended if you created your API using Anypoint Service Mesh.
 
 [[bind_the_service]]
-== Bind an API Created Using Anypoint Platform
+== Bind an API referencing an API ID
```

Comment:
Perhaps "Bind an API by using a reference to its ID"?

Reply:
Perhaps "Bind an API that references an API ID"?

Reply:
Done! Thanks!